From 5f029f7b8a545d02fc29cf5f79ea0b77d1f344ed Mon Sep 17 00:00:00 2001
From: shinchiro <shinchiro@users.noreply.github.com>
Date: Sat, 28 Jan 2017 15:29:22 +0800
Subject: [PATCH] Fix compilation on mingw

Since we compile on gcc, it should use __builtin_popcount function
---
 src/common/mathutil.h | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/common/mathutil.h b/src/common/mathutil.h
index 1243d3d..d0ab1f2 100644
--- a/src/common/mathutil.h
+++ b/src/common/mathutil.h
@@ -711,9 +711,9 @@ inline uint32_t BitfieldReverse(uint32_t value)
 // Count the 1 bits.
 inline int BitCount(unsigned int bits)
 {
-#if defined(ANGLE_PLATFORM_WINDOWS)
+#if defined(_MSC_VER)
     return static_cast<int>(__popcnt(bits));
-#elif defined(ANGLE_PLATFORM_POSIX)
+#elif defined(__GNUC__)
     return __builtin_popcount(bits);
 #else
 #error Please implement bit count for your platform!
-- 
2.10.2

